aten/src/ATen/native/quantized/cpu/qnnpack/test/fully-connected-sparse.cc

```diff
@@ -10,194 +10,135 @@
+    unit_batch_with_qmin_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(1) \
+      .inputChannels(23) \
+      .outputChannels(19) \
+      .qmin(128) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+} \
+ \
+TEST(FULLY_CONNECTED_OP_##ROW_BS ## x ##COL_BS, \
+    unit_batch_with_qmax_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(1) \
+      .inputChannels(23) \
+      .outputChannels(19) \
+      .qmax(128) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+} \
+ \
+TEST(FULLY_CONNECTED_OP_##ROW_BS ## x ##COL_BS, \
+    unit_batch_with_input_stride_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(1) \
+      .inputChannels(23) \
+      .inputStride(28) \
+      .outputChannels(19) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+} \
+ \
+TEST(FULLY_CONNECTED_OP_##ROW_BS ## x ##COL_BS, \
+    unit_batch_with_output_stride_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(1) \
+      .inputChannels(23) \
+      .outputChannels(19) \
+      .outputStride(29) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+} \
+ \
+TEST(FULLY_CONNECTED_OP_##ROW_BS ## x ##COL_BS, \
+    small_batch_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(12) \
+      .inputChannels(23) \
+      .outputChannels(19) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+} \
+ \
+TEST(FULLY_CONNECTED_OP_##ROW_BS ## x ##COL_BS, \
+    small_batch_with_qmin_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(12) \
+      .inputChannels(23) \
+      .outputChannels(19) \
+      .qmin(128) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+} \
+ \
+TEST(FULLY_CONNECTED_OP_##ROW_BS ## x ##COL_BS, \
+    small_batch_with_qmax_dynamic_prepacked) { \
+  FullyConnectedSparseOperatorTester() \
+      .batchSize(13) \
+      .inputChannels(23) \
+      .outputChannels(19) \
+      .qmax(128) \
+      .iterations(3) \
+      .rowBlockSize(ROW_BS) \
+      .colBlockSize(COL_BS) \
+      .testQ8_prepacked(FullyConnectedSparseOperatorTester::Mode::Dynamic); \
+}
+
+SPARSE_OP_TEST(1, 4)
+#if CPUINFO_ARCH_ARM || CPUINFO_ARCH_ARM64
+SPARSE_OP_TEST(8, 1)
```

Comment:
This will run for arm32 and arm64, but (1, 4) for all platforms, right?

Reply:
Yes, because I did not implement (8, 1) pattern for SSE2.